spark/src/main/scala/io/delta/sharing/spark/DeltaSharingOptions.scala

```diff
@@ -155,6 +162,10 @@ object DeltaSharingOptions extends Logging {
   val MAX_FILES_PER_TRIGGER_OPTION = "maxFilesPerTrigger"
   val MAX_FILES_PER_TRIGGER_OPTION_DEFAULT = 1000
   val MAX_BYTES_PER_TRIGGER_OPTION = "maxBytesPerTrigger"
+  // a delta sharing specific parameter, used to sepcify the max number of versions to query in
+  // one rpc in a streaming job.
+  val MAX_VERSIONS_PER_RPC = "maxVersionsPerRpc"
+  val MAX_VERSIONS_PER_RPC_DEFAULT = 100
```

Comment:
Are we going to document these ?

Reply:
yeah, we should, along with the 30 seconds. let me figure out a place to document it.